sources/packages/backend/apps/db-migrations/src/migrations/1702672829350-AddColQueueConfigurationsIsActive.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AddColQueueConfigurationsIsActive1702672829350
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Add-col-queue-configurations-is-active.sql", "Queue"),
```

Comment:
I would say, to keep it simple we can remove the queue from the enum with a TODO comment.

![image](https://github.com/bcgov/SIMS/assets/54600590/c51ee70f-46ef-4e03-87ba-bc3e00d774f5)

Reply:
![image](https://github.com/bcgov/SIMS/assets/54600590/47b1459d-f278-42d3-9d19-b2d278090f16)